transform/mattermost-analytics/models/intermediate/product/nps/int_nps_feedback.sql

```diff
@@ -0,0 +1,37 @@
+-- Materializing this intermediate table for test purposes.
+{{
+    config({
+        "materialized": "table",
```

Comment:
are we going to remove this before checkin?

Reply:
Yes, this will be dropped before checking in.